src/main/com/aca/homework/week11/stopwatch/ElapsedTimeStrategy.java

```diff
@@ -0,0 +1,7 @@
+package com.aca.homework.week11.stopwatch;
+
+public interface ElapsedTimeStrategy {
+    void print(long startMillis);
```

Comment:
let's remove this method for strategy class. would be better to have exactly one method in strategy. That will help us to use lambda expressions in the future.